modules/light-clients/08-wasm/types/vm.go

```diff
@@ -72,7 +72,7 @@ func wasmInit(ctx sdk.Context, clientStore storetypes.KVStore, cs *ClientState,
 	}
 	_, err = initContract(ctx, clientStore, cs.CodeHash, encodedData)
 	if err != nil {
-		return errorsmod.Wrapf(err, "call to wasm contract failed")
+		return errorsmod.Wrapf(ErrWasmContractCallFailed, "call to wasm contract failed: %v", err)
```

Comment:
maybe we don't need to wrap the error? it'll end up being `"call to wasm contract failed: "wasm contract call failed"`

Reply:
yea, this is mostly for anyone consuming the API, I think having a defined error (instead of whatever error a contract might return) could be helpful for whomever is eventually receiving it.